worker/uniter/remotestate/watcher.go

```diff
@@ -300,6 +299,12 @@ func (w *RemoteStateWatcher) loop(unitTag names.UnitTag) (err error) {
 		observedEvent(&seenLeadershipChange)
 	}
 
+	// TODO(wallyworld) - listen for changes to this value
```

Comment:
No, we need to properly add apis for listening to well defined model config changes. All we have now is something to listen to *any* model config change. That would kill the system if lots of uniters had to be notified about any model change.